src/Sulu/Bundle/SnippetBundle/Controller/SnippetAreaController.php

```diff
@@ -130,11 +131,11 @@ public function cgetAction(Request $request)
             $dataList[$key] = $areaData;
         }
 
-        \ksort($dataList);
+        $dataList = SortUtils::sortLocaleAware($dataList, $this->getUser()->getLocale(), fn ($a) => $a['title']);
 
         $data = [
             '_embedded' => [
-                'areas' => \array_values($dataList),
+                'areas' => $dataList,
```

Comment:
As the util function keep the index we should reintroduce the array_values part:

```suggestion
                'areas' => \array_values($dataList),
```